sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/WriteTables.java

```diff
@@ -114,8 +104,6 @@ public void processElement(ProcessContext c) throws Exception {
           tableReference, tableDestination.getTableDescription());
     }
 
-    String tempFilePrefix = resolveTempLocation(
-        c.getPipelineOptions().getTempLocation(), "BigQueryWriteTemp", stepUuid);
```

Comment:
It was only being used to get the appropriate `IOChannelFactory`, but with `FileSystems` that is computed from the file names directly afaik.